pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -285,9 +285,7 @@ message CommandPong {
 
 message CommandConsumerStats {
         required uint64 request_id         = 1;
-        required string topic_name         = 2;
-        required string subscription_name  = 3;
-        required uint64 consumer_id        = 4;
+        required uint64 consumer_id        = 2;
```

Comment:
I think a new client can prevent requesting old broker by introducing version V8, but as we already had one release 1.17 with old -client which can send `topic_name=2` to new broker so, to handle this scenario we can keep `consumer_id=4` and comment-out removed fields for future reference.